development/tests/integration/alerting/sentinel-backlog.spec.js

```diff
@@ -0,0 +1,42 @@
+const prometheus = require('../utils/prometheus.js');
+const grafana = require('../utils/grafana.js');
+const { expect } = require('chai');
```

Comment:
```suggestion
```

It is not necessary to add this, the test will understand that the expect is from chai

Reply:
Hmm, I think the only reason this works with the cht-core e2e tests is because of [this line](https://github.com/medic/cht-core/blob/077498e5eaad6027e9061e6951c513741d1ce119/tests/e2e/default/wdio.conf.js#L236) in the wdio config that makes `chai.export` available globally.

If everyone else wants it, I am happy to replicate something similar in this repo, but honestly for me I am not a big fan of the pattern of settings things globally since I think it can be confusing and obscure what is actually going on....  So, @tatilepizs  @m5r let me know what you think!

Reply:
Oh ok, that is fine, it is better to leave it then.
I removed the line and I was still able to see that the `expect` was recognized as a chai method, that is why I thought that it was safe to remove the `require`

Reply:
+1 for keeping it this way